topology/enhancers/user_metadata.go

```diff
@@ -20,7 +20,7 @@
  *
  */
 
-package metadata
+package user_topology
```

Comment:
Nit: according to the [Golang docs](https://golang.org/doc/effective_go.html#names) :  By convention, packages are given lower case, single-word names; there should be no need for underscores or mixedCaps. So this could be `usertopology` for example.